core/deployment/src/main/java/io/quarkus/deployment/steps/MainClassBuildStep.java

```diff
@@ -36,10 +36,7 @@
 import io.quarkus.gizmo.MethodDescriptor;
 import io.quarkus.gizmo.ResultHandle;
 import io.quarkus.gizmo.TryBlock;
-import io.quarkus.runtime.Application;
-import io.quarkus.runtime.StartupContext;
-import io.quarkus.runtime.StartupTask;
-import io.quarkus.runtime.Timing;
+import io.quarkus.runtime.*;
```

Comment:
I thought the formatter was rejecting `*` imports... still maybe should be fixed anyway.

Reply:
Ah, my linux machine must not have the formatter set up properly. I will fix.